common/templates/general.go

```diff
@@ -258,6 +258,14 @@ func CreateMessageSend(values ...interface{}) (*discordgo.MessageSend, error) {
 		case "filename":
 			// Cut the filename to a reasonable length if it's too long
 			filename = common.CutStringShort(ToString(val), 64)
+		case "reply":
+			msgID := ToInt64(val)
+			if msgID <= 0 {
+				return nil, errors.New(fmt.Sprintf("invalid message id '%s' provided to reply.", ToString(val)))
+			}
```

Comment:
This is not common practice within template functions; typically, we cast and let it error later. (Additionally, if we ever make `sendMessage` return API errors as is being suggested, this check will be redundant.) At any rate, if you would prefer to keep this, I would suggest writing it thusly:
```suggestion
			if msgID <= 0 {
				return nil, fmt.Errorf("invalid message reference ID %d", val)
			}
```

Reply:
Why `%d`? `val` might not be an int.

Reply:
Good point, I overlooked that. Edited.